EpikCord/guild.py

```diff
@@ -0,0 +1,317 @@
+import datetime
+from .application import Application
+from .channel import GuildChannel, WelcomeScreen, GuildStageChannel, _figure_out_channel_type, Overwrite
```

Comment:
We don't need to import `_figure_out_channel_type` now we have `Client.util`

Reply:
What am i supposed to do? Its a thing a function needs but how do we remove it?

Reply:
Replace it with `Client`.`util`.`channel_from_type` or whatever I called it.

Reply:
![image](https://user-images.githubusercontent.com/64660183/157981702-62294cb8-0c26-4f66-84f9-bb07927d6c22.png)

Reply:
How to remove?

Reply:
Like:
```py
self.client.utils._figure_out_channel_type
```
?